docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc

```diff
@@ -30,9 +30,7 @@ A component type that allows to define a container image-based configuration of
        endpoints:
          - name: maven-server
            targetPort: 3101
-           configuration:
-             protocol: tcp
-             scheme: http
+             protocol: https
```

Comment:
1. Point 1 snippet for endpoints is wrong - There is no discoverable, public property etc. Please refer to the other PR comment - https://github.com/devfile/docs/pull/43#discussion_r594567540

```
endpoints:
  - name: maven-server
    targetPort: 3101
    protocol: https
    secure: 'true'
    exposure: public
```

Reply:
Remove the extra indent as show in Maysun's example above.